app/build.gradle

```diff
@@ -152,8 +152,8 @@ dependencies {
     implementation 'com.journeyapps:zxing-android-embedded:3.6.0'
 
     //Navigation
-    implementation 'android.arch.navigation:navigation-fragment:1.0.0-rc01'
-    implementation 'android.arch.navigation:navigation-ui:1.0.0-rc02'
+    implementation 'android.arch.navigation:navigation-fragment-ktx:1.0.0-rc01'
+    implementation 'android.arch.navigation:navigation-ui-ktx:1.0.0-rc02'
```

Comment:
Safe args plugin is already added. Why this?

Reply:
Using the KTX version gives me access to the `navArgs`delegate. It's a lot cleaner to do it like this, and this way the arguments are evaluated lazily.

Reply:
OK